postgres/postgres_test.go

```diff
@@ -173,7 +174,3 @@ func Test_SslRequiredMode(t *testing.T) {
 		SslMode:  "require",
 	})
 }
-
-func Test_Postgres_Close(t *testing.T) {
```

Comment:
the store still has a close method, what reason is there for you to remove the test? please adjust it to the new conditions